src/main/java/io/strimzi/kafka/bridge/tracing/OpenTelemetryHandle.java

```diff
@@ -90,8 +91,11 @@ private SpanBuilder getSpanBuilder(RoutingContext routingContext, String operati
 
     @Override
     public <K, V> void handleRecordSpan(ConsumerRecord<K, V> record) {
-        String operationName = record.topic() + " " + MessageOperation.RECEIVE;
+        String operationName = record.topic() + " " + MessageOperation.RECEIVE.name().toLowerCase(Locale.ROOT);
```

Comment:
What is the motivation behind using `Locale.ROOT`? I do not think we use it in many other places.

Reply:
It's the way the official interceptor applies the lowecase, I was just applying the same.